justfile

```diff
@@ -130,8 +130,8 @@ purge-tests:
 gen-report:
   allure generate tests/allure/results --clean -o tests/allure/reports && cp tests/allure/reports/history/* tests/allure/results/history/. && allure open tests/allure/reports
 
-run-tests target='default' kind='kind-multi':
-  helm test {{TESTS_RELEASE}} -n {{NAMESPACE}} --kube-context kind-{{kind}} --timeout {{TESTS_TIMEOUT}} --logs
+run-tests target='default' context='kind-kind-multi':
```

Comment:
I like this change, but can you please update the other commands if you change it here?

Reply:
kube-context is used only in this `run-tests` command. There are no additional commands require context.